src/routes/posts/+page.svelte

```diff
@@ -3,10 +3,73 @@
 	import { parseContent } from '$lib/parse';
 
 	export let data;
+
+	$: category = data.category ?? null;
+	$: start = data.start ?? null;
+	$: end = data.end ?? null;
```

Comment:
I think these should just be `let` variables. I'm noticing this behavior when these are reactive. Changing to `let` fixed it for me.

https://github.com/michigandaily/alt-text-tracker/assets/44459660/131d59bf-d31d-4d58-919f-6460383b75e2